python/ql/src/Expressions/WrongNumberArgumentsForFormat.qhelp

```diff
@@ -14,7 +14,7 @@ arguments on the right hand side of the expression. Otherwise, a <code>TypeError
 
 </p></recommendation>
 <example>
-<p>In the following example the right hand side of the formatting operation can be of length 2, which does not match the format string&lt;./p>
+<p>In the following example the right hand side of the formatting operation can be of length 2, which does not match the format string.</p>
 </p><sample src="WrongNumberArgumentsForFormat.py" />
```

Comment:
It looks as if this might be the problem, and also the reason why your fix triggered the error 😄 
```suggestion
<sample src="WrongNumberArgumentsForFormat.py" />
```

Before you fixed the first end tag, it was "duplicated" - one of the few situations where "two wrongs made a right".